package.json

```diff
@@ -31,8 +31,10 @@
 		"test:e2e:linkRewrites:loadData": "hc-tools ./scripts/docs-content-link-rewrites/generate-page-paths-for-repo-and-base-path.ts",
 		"test:e2e:linkRewrites": "npx playwright test --project docs-content-link-rewrites",
 		"test:watch": "jest --watch",
-		"test": "jest",
-		"test:ci": "jest --runInBand",
+		"test:cjs": "jest",
+		"test:esm": "TEST_ESM=true NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
The updated `rehype-sanitize` is ESM-only. To run tests on it, we use a separate command, with our Jest config updated to toggle its "mode" based on `TEST_ESM`. This is cribbed from the approach in `web-platform-packages` (thanks Kevin for pointing me to that! 🙌 )